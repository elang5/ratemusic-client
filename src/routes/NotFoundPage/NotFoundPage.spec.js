import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFoundPage from './NotFoundPage'

describe(`NotFoundPage Component`, () => {
  it(`renders the NotFoundPage component`, () => {
    const wrapper = shallow(<NotFoundPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})