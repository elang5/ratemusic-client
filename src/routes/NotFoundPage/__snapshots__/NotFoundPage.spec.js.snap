// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component renders the NotFoundPage component 1`] = `
<section
  className="not-found-page"
>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</section>
`;
